SQL/0000-00-02-Modules.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE `modules` (
+  `Name` varchar(255) NOT NULL,
```

Comment:
since there's nothing referencing it at the moment, I defined the Name as the primary key. It can be revisited if we end up with foreign keys to the table later.